packages/expo-ui/ios/Menu/MenuView.swift

```diff
@@ -0,0 +1,37 @@
+// Copyright 2025-present 650 Industries. All rights reserved.
+
+import SwiftUI
+import ExpoModulesCore
+
+struct MenuView: ExpoSwiftUI.View {
```

Comment:
```suggestion
internal struct MenuView: ExpoSwiftUI.View {
```

explitly internal or just make it public (as we now opened some public views)